src/truefoundry_sdk/agent_skill_versions/raw_client.py

```
@@ -0,0 +1,444 @@
+                _items = _parsed_response.data
+                _has_next = True
+                _get_next = lambda: self.list(
+                    fqn=fqn,
+                    agent_skill_id=agent_skill_id,
+                    ml_repo_id=ml_repo_id,
+                    name=name,
+                    version=version,
+                    offset=offset + len(_items or []),
+                    limit=limit,
+                    request_options=request_options,
+                )
```

Ruff (0.15.9)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `_get_next` as a `def`

(E731)